src/styles/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$font-primary: 'Roboto', sans-serif;
+$color-white: rgba(255, 255, 255, 1);
+$color-border: rgba(243, 243, 243, 1);
+$color-botton: rgba(0, 172, 220, 1);
```

Comment:
There is a typo in the variable name '$color-botton'. It seems like it should be '$color-button'. Ensure that this variable name is consistent with its usage throughout the stylesheets.